server/src/main/java/org/atmosphere/nettosphere/NettyWebSocket.java

```diff
@@ -146,7 +146,6 @@ public void close() {
 
     public void recycle() {
         if (headers != null) headers.clear();
-        channel = null;
```

Comment:
I'm not sure about the purpose of setting channel to null here. But it doesn't look correct as it may cause NPE when calling other methods in NettyWebSocket. Probably we need just to add null checks?

Reply:
Yes, I would prefer adding null check. Thanks for the contribution!